modules/distribution/src/bin/openssl-tls.sh

```diff
@@ -176,9 +176,15 @@ PRGDIR=$(dirname "$PRG")
 # Only set CARBON_HOME if not already set
 [ -z "$CARBON_HOME" ] && CARBON_HOME=$(cd "$PRGDIR/.." ; pwd)
 
+# Check if $CARBON_HOME has spaces
+if echo "$CARBON_HOME" | grep -q ' '; then
+    echo "[Error]: Please make sure the path CARBON_HOME does not contain spaces."
```

Comment:
This message is not clear as the user will be not aware of CARBON_HOME. Lets use a generic message.